src/PlusDataCollection/OpenHaptics/vtkPlusOpenHapticsDevice.h

```diff
@@ -0,0 +1,104 @@
+/*=Plus=header=begin======================================================
+  Program: Plus
+  Copyright (c) Laboratory for Percutaneous Surgery. All rights reserved.
+  See License.txt for details.
+=========================================================Plus=header=end*/

```

Comment:
Please feel free to add your authorship information into the copyright area!

This is great work and much appreciated!